.vscode/launch.json

```diff
@@ -5,12 +5,28 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "type": "pwa-chrome",
+      "type": "node",
       "request": "launch",
-      "name": "Launch Chrome against localhost",
-      "url": "http://localhost:3000",
-      "webRoot": "${workspaceFolder}",
-      "preLaunchTask": "Start development server"
+      "name": "Jest All GDevelop.js",
+      "program": "${workspaceFolder}/GDevelop.js/node_modules/.bin/jest",
+      "args": ["--runInBand"],
```

Comment:
Interesting! Is this required/recommended?

Reply:
I used what was recommended in https://github.com/microsoft/vscode-recipes/tree/master/debugging-jest-tests